### PR TITLE
refac: 2. refactor building blocks to clean architecture

### PR DESCRIPTION
**Pull request by AndersBallingPetersen**
## Description
This module does not need an Application layer.

image
## Checklist
- [ ] Should the change be behind a feature flag?
- [ ] Can the feature be meaningfully disabled or circumvented if there are issues (e.g., database-breaking changes)?
- [ ] Has it been considered whether data is being delivered to the wrong actor?
- [ ] Subsystem test executed (dev_002/dev_003)
- [ ] Is there time to monitor state of the release to Production?
- [ ] Reference to the task

### CODE REVIEW

File: source/BuildingBlocks.Application/BuildingBlocks.Application.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (MadsDue) [inline source/BuildingBlocks.Application/BuildingBlocks.Application.csproj:1]: This one is missing
AssemblyName
RootNamespace

File: source/BuildingBlocks.Application/BuildingBlocks.Application.csproj [hunk 2]
```diff
@@ -1,33 +1,17 @@
...
-    </ItemGroup>
-
-    <ItemGroup>
-      <ProjectReference Include="..\BuildingBlocks.Infrastructure\BuildingBlocks.Infrastructure.csproj" />
-    </ItemGroup>
-
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>Energinet.DataHub.EDI.BuildingBlocks.Interfaces</AssemblyName>
+    <RootNamespace>Energinet.DataHub.EDI.BuildingBlocks.Interfaces</RootNamespace>
```
Comment (MadsDue) [inline source/BuildingBlocks.Application/BuildingBlocks.Application.csproj:4]:
```suggestion
    <RootNamespace>Energinet.DataHub.EDI.BuildingBlocks.Application</RootNamespace>
```

File: source/BuildingBlocks.Interfaces/BuildingBlocks.Interfaces.csproj [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
     <ItemGroup>
+        <PackageReference Include="Azure.Core" Version="1.44.0" />
         <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="17.11.20">
           <PrivateAssets>all</PrivateAssets>
           <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
     </ItemGroup>
 
+    <ItemGroup>
+      <ProjectReference Include="..\BuildingBlocks.Domain\BuildingBlocks.Domain.csproj" />
```
[the remark this answers is not in the document]
Comment (AndersBallingPetersen) [reply source/BuildingBlocks.Interfaces/BuildingBlocks.Interfaces.csproj:16]: Only in BuildingBlocks

**Review by MadsDue — DISMISSED** [review]
LGTM

**Review by MadsDue — APPROVED** [review]

**Pull request merged**